stockfish/models.py

```diff
@@ -514,18 +514,28 @@ def get_evaluation(self) -> dict:
             elif splitted_text[0] == "bestmove":
                 return evaluation
 
-    def get_top_moves(self, num_top_moves: int = 5) -> List[dict]:
+    def get_top_moves(
+        self, num_top_moves: int = 5, include_info: bool = False
+    ) -> List[dict]:
         """Returns info on the top moves in the position.
 
         Args:
             num_top_moves:
                 The number of moves to return info on, assuming there are at least
                 those many legal moves.
 
+            include_info:
+                Option to include the full info from the engine in the returned dictionary,
+                including seldepth, multipv, time, nodes, nps, and wdl if available.
+                Boolean. Default is False.
+
         Returns:
             A list of dictionaries. In each dictionary, there are keys for Move, Centipawn, and Mate;
             the corresponding value for either the Centipawn or Mate key will be None.
             If there are no moves in the position, an empty list is returned.
+
+            If include_info is True, the dictionary will also include the keys SelectiveDepth, Time,
+            Nodes, N/s, MultiPVLine, and WDL (if available). WDL is set from the White player's perspective.
```

Comment:
> WDL is set from the White player's perspective.

I would actually like to keep the return value from this function consistant to that of `get_wdl_stats()`. But the other function returns the stats as list of three integers (which actually makes more sense to me for this) and from the point of view of the person whose turn it is. For the latter, I don't really know which makes more sense, but now that we have implemented it this way, it would probably make sense to keep it for backwards compatibility reasons.
@johndoknas what's your opinion on that?

Reply:
I believe WDL is set from White player's perspective from Stockfish. 

Personally, I think that makes sense: regular +/- notation is also set from White's perspective (ie. it's not +4 when Black is winning and it's their turn). It also makes for easier reading: no need for extra information on whose turn it is to read WDL. 

As for consistency, I agree it's unfortunate to have different outputs. Perhaps a flag in `get_wdl_stats()` could help, eg. `get_wdl_stats(white_perspective=False)` for backwards compatibilty?

Reply:
For the Stockfish engine, I think it returns evaluation for the side to move, both for centipawn and wdl. E.g., the get_wdl_stats function just reads the stockfish executable output directly. For many common programs that use stockfish though (lichess, chess.com, etc), they display evaluation from White's perspective.

Personally I'm fine with either, but maybe for maintaining consistency we should default to the side to move. An optional parameter would be fine though (both for get_top_mvoes and get_wdl_stats).

Reply:
Yes, sorry, confirmed: Stockfish returns eval for the side to move, both for WDL and cpl. 

OK, agreed on default "side to move" for consistency. Will update.